### PR TITLE
docs: add ng-mocks to tooling resources

### PR DESCRIPTION
**Pull request by satanTime**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## Other information

Adding info about a testing library `ng-mocks`.

- https://github.com/ike18t/ng-mocks
- https://www.npmjs.com/package/ng-mocks

### CODE REVIEW

**Comment by satanTime** [comment]
@googlebot I signed it!

File: aio/content/marketing/resources.json [hunk 1]
```diff
@@ -282,6 +282,12 @@
             "title": "Scully",
             "logo": "https://raw.githubusercontent.com/scullyio/scully/main/assets/logos/PNG/Green/scullyio-logo-green.png",
             "url": "https://scully.io"
+          },
+          "ng-mocks": {
+            "desc": "A testing library for creating mock services, components, directives, pipes and modules out of annoying dependencies in unit tests.",
+            "title": "ng-mocks",
+            "logo": "",
+            "url": "https://github.com/ike18t/ng-mocks"
```
Comment (gkalpak) [inline aio/content/marketing/resources.json:290]: FYI, you could append `#readme` to the URL and have the page scroll automatically to the README when visiting:

```suggestion
            "url": "https://github.com/ike18t/ng-mocks#readme"
```

(This is just fyi - it's up to you what URL to use.)

**Review by gkalpak — APPROVED** [review]
Thx, @satanTime :+1:

Could you please update the commit message header to something like (as it makes it easier to know what the commit is about when scanning over the commit history):
> docs: add ng-mocks to tooling resources

File: aio/content/marketing/resources.json (hunk 1 above)
Comment (satanTime) [reply aio/content/marketing/resources.json]: done

**Comment by satanTime** [comment]
> Thx, @satanTime 👍
> 
> Could you please update the commit message header to something like (as it makes it easier to know what the commit is about when scanning over the commit history):
> 
> > docs: add ng-mocks to tooling resources

done

**Review by mgechev — APPROVED** [review]

**Pull request closed without merging**